packages/features/ee/workflows/components/WorkflowDropdownFilter.tsx

```diff
@@ -0,0 +1,100 @@
+import { useSession } from "next-auth/react";
+import type { ReactNode } from "react";
+import { useState } from "react";
+
+import { useFilterQuery } from "@calcom/features/bookings/lib/useFilterQuery";
+import { classNames } from "@calcom/lib";
+import { getPlaceholderAvatar } from "@calcom/lib/defaultAvatarImage";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { AnimatedPopover, Avatar } from "@calcom/ui";
+import { Layers, User, Check } from "@calcom/ui/components/icon";
+
+export const WorkflowDropdownFilter = () => {
+  const { t } = useLocale();
+  const session = useSession();
+
+  const [dropdownTitle, setDropdownTitle] = useState<string>(t("all_apps"));
+  const { data: query, pushItemToKey, removeItemByKeyAndValue, removeAllQueryParams } = useFilterQuery();
+  const { data: teams } = trpc.viewer.teams.list.useQuery();
```

Comment:
will change this query after sean's branch has been merged into feat/organizations